src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -157,6 +159,38 @@ public SpeechletResponse onIntent(IntentRequest request, Session session)
             return setStopNumber(
                     stopNumberStr,
                     session);
+        } else if (YES.equals(intent.getName())) {
+            ArrayList<ObaStop> stops = (ArrayList<ObaStop>) session.getAttribute(FOUND_STOPS);
```

Comment:
I would suggest moving the code for the YES and NO intents into their own method, and also include a session variable defining what the user is answering yes or no to.  We may end up using YES and NO for something else in the future, and it would be nice to already have that logic structure set up.